index.js

```diff
@@ -605,6 +605,11 @@
            other.unicode === this.unicode;
   }
 
+  //  RegExp$prototype$lte :: RegExp ~> RegExp -> Boolean
+  function RegExp$prototype$lte(other) {
+    return lte(toString(this), toString(other));
+  }
```

Comment:
@davidchambers I had originally implemented `lte` for `RegExp` as well but couldn't come up with a satisfying semantic for ordering on flags. Just doing `String` comparison seems like it is avoiding the issue.